lib/content-services/src/lib/content-metadata/components/content-metadata/content-metadata.component.ts

```diff
@@ -144,7 +169,10 @@ export class ContentMetadataComponent implements OnChanges, OnInit, OnDestroy {
         private translationService: TranslationService,
         private appConfig: AppConfigService,
         private tagService: TagService,
-        private categoryService: CategoryService
+        private categoryService: CategoryService,
+        private cdr: ChangeDetectorRef,
+        private contentService: ContentService,
+        private snackBar: MatSnackBar
```

Comment:
Could you please use NotificationService instead of MatSnackBar? NotificationService uses MatSnackBar too but it displays custom component as snackbar so for example it will avoid some problems like accessibility problems which we fixed as part of custom component